src/Uno.UI.FluentTheme.v1/XamlControlsResourcesV1.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+using Uno;
+using Uno.Extensions;
+using Uno.UI.Xaml;
+using Windows.UI.Xaml;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	public sealed partial class XamlControlsResourcesV1 : ResourceDictionary
```

Comment:
Is it possible for this to inherit from `XamlControlsResources`? I am thinking about code-behind which eg may look for that type in Application.Resources.

Reply:
Unfortunately, referencing `XamlControlsResources will force an unconditional reference to v1 and v2, and make the linker keep both versions.